src/utils/Bootstrap.cpp

```diff
@@ -99,34 +103,89 @@ int Bootstrap::init(int argc, char** argv)
     llvm::InitializeNativeTarget();
```

Comment:
I included namespace `llvm` because I was lazy to write `llvm::` all the time :P, however, thank you for pointing that, as always.. (y).. I will fix it shortly